docs/cli/odigos_install.mdx

```diff
@@ -40,7 +40,7 @@ odigos install [flags]
         --odiglet-image             odiglet container image name
         --instrumentor-image        instrumentor container image name
         --autoscaler-image          autoscaler container image name
-        --image-prefix              prefix for all container images. used when your cluster doesn't have access to docker hub
+        --image-prefix              prefix for collector images. Used when your cluster doesn't have access to docker hub or when a different base collector image is needed. 
```

Comment:
```suggestion
        --image-prefix              prefix for collector images. Used when your cluster doesn't have access to docker hub. 
```

I think it's not feasible at the moment to support other base images, due to the coupling of the autoscaler with the custom processors/exporters and the version of components